### ISSUE TITLE
Homepage v2: Marketplaces preview component

### ISSUE THREAD

**Issue by austinmilt** [holaplex/holaplex#333]
Create a new component to preview a single marketplace

## Default Styling
![image](https://user-images.githubusercontent.com/1540936/163828129-995fe80e-d62c-45d1-90de-4d07a4bdb47b.png)

- preview image is marketplace banner image
- top-left icons are creator profile icons (see usage of `Avatar` and `OverlappingCircles` in [address].tsx)
- lower-left text is the name of the marketplace

## Hover State
![image](https://user-images.githubusercontent.com/1540936/163828478-cd7f4c07-5ed3-492d-ab88-da6d42a02f14.png)

- on hover, show number of NFTs in marketplace in lower-left
- on hover, show floor price in lower-right

**Comment by austinmilt** [comment]
@bellebellesol @damianr 
Is there any special interaction behavior other than what's described here?

**Comment by bellebellesol** [comment]
no, for now this is good. it might be worth adding a subtle grow interaction on hover like we do for NFT cards on the home page currently. @damianr what do you think?

**Comment by bellebellesol** [comment]
small change: lets make the text shown on hover and by default on mobile white instead of grey-300 for better readability:

<img width="301" alt="Screen Shot 2022-04-18 at 4 59 10 pm" src="https://user-images.githubusercontent.com/95664179/163836297-15bb398c-0bd1-446a-bf13-e09ad22e653d.png">
